charts/ccd-data-store-api/Chart.yaml

```diff
@@ -2,13 +2,13 @@ description: Helm chart for the HMCTS CCD Data Store
 name: ccd-data-store-api
 apiVersion: v2
 home: https://github.com/hmcts/ccd-data-store-api
-version: 1.7.2
+version: 1.7.3
 maintainers:
   - name: HMCTS CCD Dev Team
     email: ccd-devops@HMCTS.NET
 dependencies:
   - name: java
-    version: 3.4.4
+    version: 3.4.2
```

Comment:
Doesn't look right that we're downgrading? 🤔

Reply:
That is right. good spot. it was committed but was not pushed it. all done.

Reply:
It should be down down graded